docs/rmis.md

```diff
@@ -0,0 +1,243 @@
+ customization
+
+## Interoperability Requirements
+
+ In which formats and structures should data be stored in the R / MIS to allow for an exchange with others relevant to the pro- ject / organization? In general, the R / MIS must fit into the already
+ existing IT architecture and with the existing IT knowledge of the PEA
+ and country. The following data standards should be consid- ered as
+ part of the setup of any R / MIS:
+
+-   Consider (project-specific) required file formats (e.g., CSV, XLS,
+    KML, JSON) or Application Programming Interfaces (API) for data
+    import and export
+-   KfW requires the export of geo-referenced project data (in KML or
+    XLS) according to a new template (see below) as part of its
+    regular reporting. This template also ensures compatibility of the
+    reporting with the International Aid Transparency Initiative (IATI)
+    Standard.
+-   The IATI standard is an XML-based set of rules and guidance for
+    publishing data of development and humanitarian organiza- tions to
+    improve their coordination, accountability, and effec- tiveness and
+    to increase the transparency of information on resources flowing
+    into developing countries. All BMZ-funded projects must provide
+    IATI-compatible data to KfW.
+-   Open Data Kit (ODK) is an XLS-based standard format for exchanging
+    data between R / MIS, mobile data collection, and other compatible
+    tools.
+-   Sector-specific open data standards also must be considered, if they
+    are applicable (e.g., EPA standard for environmental data;
+    [www.epa.gov/data-standards](http://www.epa.gov/data-standards)).
+
+## Legal Aspects
+
+ **Data Security:** If a desktop software system is used, data secu-
+ rity should be warranted by ensuring:
+
+ that the data are protected against unauthorized or accidental
+ destruction, accidental loss, technical defects, falsification, theft,
+ unlawful use, unauthorized modification or duplication, and other
+ forms of unauthorized access and use by technical means regular backup of data appropriate security of hardware
+
+ If the software is provided as web-based service (SaaS), the provider should be contractually obliged to take appropriate organizational and technical precautions to prevent disruptions to the
+ availability, integrity, authenticity, and confidentiality of their IT
+ systems, components, and processes essential to the functioning of the
+ critical infrastructures they operate. The provider should implement
+ data backup concepts, ensuring the protection of data against loss due
+ to system-related failures.
+
+**Data protection:** Wherever possible, the collection of personal data by
+ the R / MIS-owner (data controller) should be avoided. If this is
+ impossible, the main principles of data protection must be adhered to,
+ that is, personal data shall:
+
+ be processed lawfully, fairly, and in a transparent manner in relation
```

Comment:
Suggestion: How about we include bullets here in every single point after "personal data shall:" in this way it seems nicely structured and a little bit clearer? This includes each line down to 193.